experimental/dds/tree2/package.json

```diff
@@ -37,7 +37,7 @@
 		"prettier": "prettier --check . --ignore-path ../../../.prettierignore",
 		"prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
-		"test:benchmark:report": "mocha --exit --perfMode --parentProcess --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 60000",
+		"test:benchmark:report": "mocha --node-option unhandled-rejections=strict,expose-gc --exit --perfMode --parentProcess  --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 60000",
```

Comment:
I think these options come from the moch config and were removed on purpose, so I think this is also a regression. Perhaps a bad merge?